packages/ng/src/app/select/api/select-api.component.ts

```diff
@@ -29,7 +30,7 @@ import { ISelectApiFeeder } from './picker';
 	],
 })
 // tslint:disable-next-line:component-class-suffix
-export class LuSelectApi<T>
+export class LuSelectApi<T extends IApiItem = IApiItem>
```

Comment:
I'm not agree with this, because, the LuSelectApi should as generic as possible, By forcing a cast in IApiItem, you force a kind of api. So for RDDFeeder, it makes sens but not for generic LuSelect API, can you please remove this `extends`